### ISSUE TITLE
Captions is not giving accurately for nature images and scenaries

### ISSUE THREAD

**Issue by rakashi** [tensorflow/models#2767]
Hi,

I have trained model using MSCOCO dataset with inception pretrained checkpoints. I have got the perplexity as 10.0345. I am considering this as 80% of validation accuracy. After training completion I have tested the model but it is not giving the captions accurately. So now I want to improve the model accuracy. Is MSCOCO dataset is enough to get the captions or any other dataset are available.

And Below I have attached some images of what I have tested with my trained model So I tested with some images the out put also I am attaching below.

The model is not detecting any animals present in the image. So How can I improve the model accuracy and correct predictions.

can anyone suggest what I have to do to improve the accuracy

![screenshot from 2017-11-10 17 29 13](https://user-images.githubusercontent.com/30826794/32657702-b26ea872-c63c-11e7-8509-8721a543abbe.png)

I have compared with microsoft captions with my models(25000 epochs and 1,00,000 epochs
![screenshot from 2017-11-10 17 25 31](https://user-images.githubusercontent.com/30826794/32657624-71f569de-c63c-11e7-8b21-870021b3522a.png)
)

**Comment by angersson** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
